controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -71,7 +71,7 @@ type StorageConsumerReconciler struct {
 // Note:
 // The Controller will requeue the Request to be processed again if the returned error is non-nil or
 // Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
-func (r *StorageConsumerReconciler) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
+func (r *StorageConsumerReconciler) Reconcile(_ context.Context, request reconcile.Request) (reconcile.Result, error) {
```

Comment:
Lets assign this ctx value to ctx in StorageConsumerReconciler `r.ctx = ctx`

Reply:
Ack